### PR TITLE
PAYARA-1920 don't use localized description in domain.xml fixes #1827

### LINKED ISSUE #1827: MultiByteCode (charset = cp 932) is included in domain.xml.
# Description #
----------
MultiByteCode (charset = cp 932) is included in domain.xml.

in ..

&lt;domain&gt;
  &lt;configs&gt;
    &lt;config name="server-config"&gt;
      &lt;system-property name="JMS_PROVIDER_PORT" description="...([cp932] MultiByteCode)."&gt;

.and.

&lt;domain&gt;
  &lt;configs&gt;
    &lt;config name="default-config"&gt;
      &lt;system-property name="JMS_PROVIDER_PORT" description="...([cp932] MultiByteCode)."&gt;

This problem has been around since GlassFish 4.0.

## Current Outcome

1) There is no problem with standard usage
2) When trying to execute an EJB unit test using
EJBContainar.createEJBContainer() The following message appears and the execution stops.

javax.ejb.EJBException: Failed to create a temporary domain.xml file.
Use log level FINE for more details.

3) Create a new standalone server instance,
The following message will appear when starting it and it will not start.

Caused by: org.glassfish.config.support.DomainXmlPreParser$DomainXmlPreParserException:
javax.xml.stream.XMLStreamException: ParseError at [row,col]:[89,40]

It is possible to deal by deleting the corresponding part, but when domain.xml is regenerated
It is necessary to do it every time.

## Environment ##

- **Payara Version**: 4.1.1.164
- **Edition**: Full
- **JDK Version**: 1.8 U112
- **Operating System**: Windows 4
- **Database**: n/a
- **Language**: Japanease

### PR DESCRIPTION
**Pull request by smillidge**

### CODE REVIEW

File: appserver/jms/gf-jms-connector/src/main/resources/META-INF/configuration/jms-module-conf.xml [hunk 1]
```diff
@@ -43,15 +43,15 @@
...
+                description="Port Number that JMS Service will listen for remote clients connection.">
             <port base-offset="7676"></port>
             </customization-token>
         </customization-tokens>
         <configuration-element config-bean-class-name="com.sun.enterprise.connectors.jms.config.JmsService">
 <![CDATA[
     <jms-service default-jms-host="default_JMS_host" type="EMBEDDED">
         <jms-host port="${JMS_PROVIDER_PORT}" host="localhost" name="default_JMS_host"/>
-   </jms-service>
+   <!--</jms-service>-->
```
Comment (Pandrex247) [inline appserver/jms/gf-jms-connector/src/main/resources/META-INF/configuration/jms-module-conf.xml]: Isn't the _jms-service_ tag now not closed?

Comment (smillidge) [inline appserver/jms/gf-jms-connector/src/main/resources/META-INF/configuration/jms-module-conf.xml]: yeah not sure where the comment came from

**Comment by smillidge** [comment]
jenkins test please

**Comment by Pandrex247** [comment]
Ripping out the localisation seems a bit excessive - is there not a more elegant solution?

**Pull request merged**